src/main/java/core/basesyntax/service/impl/DataValidatorImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataValidator;
+
+public class DataValidatorImpl implements DataValidator {
+    @Override
+    public boolean isNotValidDataFromCsv(String[] data) {
```

Comment:
краще не йти від зворотнього і перевіряти в методі чи валідні дані
```suggestion
    public boolean isValidDataFromCsv(String[] data) {
```